compiler/crates/relay-typegen/tests/generate_typescript/fixtures/updatable-fragment-spread-multiple.expected

```diff
@@ -0,0 +1,47 @@
+==================================== INPUT ====================================
+query updatableFragmentSpreadAndRegularSpreadQuery {
+	me {
+		...updatableFragmentSpreadAndRegularSpread_updatable_user
+		...updatableFragmentSpreadAndRegularSpread_2_updatable_user
+	}
+}
+
+fragment updatableFragmentSpreadAndRegularSpread_updatable_user on User @updatable {
+	firstName
+}
+
+fragment updatableFragmentSpreadAndRegularSpread_2_updatable_user on User @updatable {
+	firstName
+}
+==================================== OUTPUT ===================================
+import { FragmentRefs } from "relay-runtime";
+export type updatableFragmentSpreadAndRegularSpreadQuery$variables = {};
+export type updatableFragmentSpreadAndRegularSpreadQuery$data = {
+  readonly me: {
+    readonly $updatableFragmentSpreads: FragmentRefs<"updatableFragmentSpreadAndRegularSpread_2_updatable_user" | "updatableFragmentSpreadAndRegularSpread_updatable_user">;
+  } | null;
+};
+export type updatableFragmentSpreadAndRegularSpreadQuery = {
+  response: updatableFragmentSpreadAndRegularSpreadQuery$data;
+  variables: updatableFragmentSpreadAndRegularSpreadQuery$variables;
+};
+-------------------------------------------------------------------------------
+import { FragmentRefs } from "relay-runtime";
+export type updatableFragmentSpreadAndRegularSpread_2_updatable_user$data = {
+  firstName: string | null;
+  readonly " $fragmentType": "updatableFragmentSpreadAndRegularSpread_2_updatable_user";
+};
+export type updatableFragmentSpreadAndRegularSpread_2_updatable_user$key = {
+  readonly " $data"?: updatableFragmentSpreadAndRegularSpread_2_updatable_user$data;
+  readonly $updatableFragmentSpreads: FragmentRefs<"updatableFragmentSpreadAndRegularSpread_2_updatable_user">;
+};
```

Comment:
In Flow this object, and  `updatableFragmentSpreadAndRegularSpread_2_updatable_user$data` above is inexact (`...`). I don't have a good sense of why, but presumably that means we need some TypeScript equivalent? Or maybe it can be removed from the Flow types?

https://flow.org/en/docs/types/objects/#exact-and-inexact-object-types

Reply:
How would such an inexact type be typed in Typescript, would it just be a union of
```ts
type inexact = original_obj & { [k: string]: unknown };
```

Might need more input from some TypeScript wizard before I add this...

Reply:
Maybe we can first start with understanding why its inexact in Flow? What breaks if we make it inexact? Is this specific to updatable fragments, or are all fragment keys inexact in Flow?

Maybe the semantics of TypeScript are different and we don't actually need them to be inexact in TypeScript.

Reply:
In other places inexact objects are used for fragments, if their MaskStatus is unmasked. I know too little about how fragments work to infer why it would be important to have the same characteristics when updating a fragment... 

I tried tracing the git log for an explanation of why an InexactObject was being used, but haven't found anything. @rbalicki2 you seem to have added the code, do you perhaps remember why the InexactObject was being used?